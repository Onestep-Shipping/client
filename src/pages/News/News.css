.body-container2 {
  width: 100%;
  height: 85%;
  display: flex;
  flex-direction: column;
  padding: 50px;
  align-items: center;
  background-color: white;
}