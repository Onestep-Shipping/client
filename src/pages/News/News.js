import React from 'react';
import './News.css';

import Header from '../../components/Header/Header.js';

const News = () => {
  return (
    <div class="homepage-container">
      <Header />
      <div class="body-container2">
        News
      </div>
    </div>
  );
};

export default News;
