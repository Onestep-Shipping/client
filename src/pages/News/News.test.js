import News from './News';
import React from 'react';
import { shallow } from 'enzyme';

let wrapper = shallow(<News />);

describe('News', () => {
  it('should render the News Component correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});