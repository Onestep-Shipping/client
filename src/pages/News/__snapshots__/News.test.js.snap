// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News should render the News Component correctly 1`] = `
<div
  className="homepage-container"
>
  <Header />
  <div
    className="body-container2"
  >
    News
  </div>
</div>
`;
